jsonnet/openebs-mixin/build.sh

```diff
@@ -26,7 +26,7 @@ generateRules(){
 	rm -rf $rulesDirPath
 	mkdir -p  $rulesDirPath
 	jsonnet -J vendor -m $rulesDirPath lib/rules.jsonnet
-	rulesFolder=(cStor Jiva Volume)
+	rulesFolder=(cStor Jiva Volume NPD)
```

Comment:
Is there an extra space between cStor and Jiva?

Reply:
No, it's fine. There is only one space present.